tests/test_cli.py

```diff
@@ -39,6 +61,39 @@ def test_good_cli_run(tmp_path, template_path):
     assert answers["_src_path"] == str(template_path)
 
 
+def test_good_cli_run_dot_config(tmp_path, template_path_with_dot_config):
+    with local.cwd(str(template_path_with_dot_config)):
+        git_commands()
+
+    run_result = CopierApp.run(
+        ["--quiet", str(template_path_with_dot_config), str(tmp_path)],
```

Comment:
You need this for the test to be correct.

```suggestion
        ["--quiet", "-a", ".config/altered-answers.yml", str(template_path_with_dot_config), str(tmp_path)],
```